index.html

```diff
@@ -0,0 +1,322 @@
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <button type="submit" class="blind">See Project</button>
+        </div>
+      </div>
+      <div class="card1">
+        <div class="snapshooti">
+          <img src="./images/snapshot.png" alt="snapshot" class="responsive" />
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Facebook 360</h2>
+            <ul class="canop">
+              <li>FACEBOOK</li>
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <button type="submit" class="see">See Project</button>
+        </div>
+      </div>
+      <div class="card2">
+        <div class="snapt">
+          <img src="./images/snapshot2.png" alt="snapshot" class="responsive">
+        </div>
+        <div class="titol">
+          <div class="w-title">
+            <h2 class="name">Uber Navigation</h2>
+            <ul class="canop">
+              <li>Uber</li>
+              <li>Lead developer</li>
+              <li>2018</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <button type="submit" class="blind">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="me">
+        <h2 class="myself">About Myself</h2>
+        <p class="hello">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div>
+          <h4 class="connect">LET'S CONNECT</h4>
+          <ul class="icon">
+            <li>
+              <a href="#"
```

Comment:
- [ ] Please reduce the gap between social media icons and make sure it matches the Figma design

### Your design

![image](https://user-images.githubusercontent.com/83803414/166201214-d357d1dc-2306-4556-bc14-ab9cc7f35896.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166201229-070b0233-b536-437b-80cb-9e87557794b1.png)